src/components/recorder/RightSidePanel.tsx

```
@@ -549,6 +578,31 @@ export const RightSidePanel: React.FC<RightSidePanelProps> = ({ onFinishCapture
   const handleConfirmListCapture = useCallback(() => {
     switch (captureStage) {
       case 'initial':
+        const hasValidListSelectorForCurrentAction = browserSteps.some(step =>
+          step.type === 'list' &&
+          step.actionId === currentListActionId &&
+          step.listSelector &&
+          Object.keys(step.fields).length > 0
+        );
+        
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)